6.3.22/script.js

```diff
@@ -0,0 +1,33 @@
+/*------------------------------------------------------*/
+const dishes = ['Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup',
+'Salmon'];
+dishes.food = "I am hungry";
+for (let i in dishes) {
+	console.log(i);
+}
+
+for (let i of dishes) {
+	console.log(i);
+}
+/*-----------------------------------------------------*/
+const btn = document.querySelector('#btn');
+btn.onclick = () => console.log('Javascript test');
+
+function clickPlease() {
+var seltheme = document.getElementById("selcolor").value;
+   var elem = document.getElementById("pict");
+   if(seltheme == "Maroon") {
+   elem.style.backgroundColor = 'red';
+   }
+   else  if(seltheme == "Green") {
+   elem.style.backgroundColor = 'green';
+   }
+};
+/*---------------------------------*/
+
+const myResult = fetch('https://reqres.in/api/users')
+.then(response => response.json())
+.then(data => data.data)
+.then(users => users.map(user => user.first_name))
+.then(names => [names[0], names[1], names[2]])
+.then(threeNames => console.log(threeNames))
```

Comment:
YOu don't need to save the result in the myResult variable, you are going to do something right away with the data
  you just need to just need the first .then to convert the data to json, and the second one to do somethign with the data, in this case we want run a for loop and get the first_name for the first 3 users.
please take a look at this : https://www.freecodecamp.org/news/javascript-fetch-api-tutorial-with-js-fetch-post-and-header-examples/#fetch-api